config/values/scholarships.yml

```diff
@@ -4,8 +4,6 @@ scholarships:
   languagesfrenchgermanspanish: £28,000
   maths: £31,000
   physics: £31,000
-  nonuk:
```

Comment:
@emmaachesongray You can't delete this scholarships variable  it is used on `/non-uk-teachers/get-an-international-relocation-payment.md` and `/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md` (that's why the unit test has failed)